src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,17 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+
+public abstract class Command {
```

Comment:
Since this is an abstract method, it would help to let other developers know what this is for. 
You now know how to write doc comments. So please incorporate them sooner than later.